hepcrawl/pipelines.py

```diff
@@ -116,8 +116,11 @@ def process_item(self, item, spider):
 
         hep_record = self._post_enhance_item(item, spider)
 
-        validate(hep_record, 'hep')
-        spider.logger.debug('Validated item by Inspire Schemas.')
+        try:
+            validate(hep_record, 'hep')
+            spider.logger.debug('Validated item by Inspire Schemas.')
+        except Exception as err:
+            spider.logger.error('ERROR in validating {}: {}'.format(hep_record, err))
```

Comment:
Make sure that this is needed, and if it is, change the `.format` by `error('message %s', message)`.

Reply:
This seemed safe to remove as I checked and scrapy does not stop unless a certain number of exceptions is raised (`CLOSESPIDER_ERRORCOUNT`) which defaults to infinity. However, I removed this and all of the records failed with `ValidationError: u'_collections' is a required property`. This seems odd as the record generation is done by DoJSON and the code looks basically the same as DESY spider. I will investigate, but maybe you know something about this.

Reply:
Nothing pops up right now, will have to check :/

Reply:
are you using a version of doJSON that includes the CDS conversion? you didn't bump the version in setup.py, so maybe you are using an older version that does no preprocessing for CDS.

Reply:
@michamos I bumped and reinstalled dependencies, I now have version 57.1.5, no change. Is there any special way I need to be using DoJSON in this case?

Reply:
after discussing IRL, turns out the new API that includes CDS conversion was not used.